src/mig/modules/scribe/doc.rst

```diff
@@ -0,0 +1,148 @@
+===================================
+Mozilla InvestiGator: scribe module
+===================================
+:Author: Aaron Meihm <ameihm@mozilla.com>
+
+.. sectnum::
+.. contents:: Table of Contents
+
+The scribe module provides host-based analysis based on a JSON document
+containing a series of tests. The module is based on the scribe engine;
+scribe can be found `here <https://github.com/mozilla/scribe>`_.
+
+The scribe module is intended to help support:
+
+* Executing policy checks on systems, for example as part of using MIG for vulnerability management
+* Execute more advanced file content tests involving dependencies
+
+In addition to supporting on-host analysis using scribe documents, the
+module can also be used to query elements of the host system such as the
+package manager database for installed package information.
+
+This document does not discuss the details around writing scribe tests, the
+scribe project documentation should be reviewed for that. This document focuses
+on usage of the scribe module within MIG and provides some examples.
+
+Usage
+-----
+The scribe module supports two execution modes, scribe document analysis
+and package query mode. To use document analysis mode, the `path` option can
+be given to the module along with the path to a scribe document. To use
+package query mode, the `pkgmatch` option is supplied with a regular expression
+specifying which packages to return information for from the agent.
```

Comment:
Since both use cases are fairly autonomous, would it make sense to make two separate modules? the `scribe` module would focus on parsing and executing a scribe logic, and the `pkg` module would only provide package querying (using scribe under the hood).